subgraphs/ens-governance/schema.graphql

```diff
@@ -0,0 +1,151 @@
+type Governance @entity {
+  "Unique entity used to keep track of common aggregated data"
+  id: ID!
+  "Number of proposals created"
+  proposals: BigInt!
+
+  "Total number of token holders currently"
+  currentTokenHolders: BigInt!
+  "Total number of token holders"
+  totalTokenHolders: BigInt!
+  "Total number of delegates participating on the governance currently"
+  currentDelegates: BigInt!
+  "Total number of delegates that held delegated votes"
+  totalDelegates: BigInt!
+  "Total number of votes delegated expressed in the smallest unit of the Compound Token"
+  delegatedVotesRaw: BigInt!
+  "Total number of votes delegated expressed as a BigDecimal normalized value for the Compound Token"
+  delegatedVotes: BigDecimal!
+
+  "Number of proposals currently queued for execution"
+  proposalsQueued: BigInt!
+  "Number of proposals currently executed"
+  proposalsExecuted: BigInt!
+  "Number of proposals currently canceled"
+  proposalsCanceled: BigInt!
+
+  "Quorum fraction numerator value. (quorum = totalSupply * numerator / denominator)"
+  quorumNumerator: BigInt
+  "Quorum fraction denominator value. (quorum = totalSupply * numerator / denominator)"
+  quorumDenominator: BigInt
+}
+
+type Proposal @entity {
+  id: ID!
```

Comment:
Should document every field. e.g. what does the ID look like? The goal is for someone be able to figure out what each field is/does just by looking at the schema in the subgraph playground